src/containers/menu/menuImage.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ImageProps } from 'react-native';
+import { ThemeKey } from '@src/core/themes';
+
+export type LayoutMenuImageConfig = { [key in ThemeKey]?: React.ReactElement<ImageProps> };
+
+export class MenuImage {
+
+  public static select = (config: LayoutMenuImageConfig,
```

Comment:
Should be geneneric which returns Generic or null

Reply:
Also makes sense to return default value if has no value found in config:
```ts
select({
  'light': foo,
  'default': bar,
});
```
Take a look on React Native's `Platform.select` function

Reply:
Done.